shared/src/store/action-reducers/emergency-operation-center.ts

```diff
@@ -15,6 +39,25 @@ export class AddLogEntryAction implements Action {
     public readonly message!: string;
 }
 
+export class SendAlarmGroupAction implements Action {
+    @IsValue('[Emergency Operation Center] Send Alarm Group' as const)
+    public readonly type = '[Emergency Operation Center] Send Alarm Group';
+    @IsString()
+    @MaxLength(255)
+    public readonly name!: string;
+    @IsUUID(4, uuidValidationOptions)
+    public readonly alarmGroupId!: UUID;
+    @IsUUID(4, uuidValidationOptions)
+    public readonly targetTransferPointId!: UUID;
+    @IsOptional()
+    @IsInt()
+    @Min(0)
+    public readonly firstVehiclesCount: number | undefined;
```

Comment:
This does not have to be optional since `0` and `undefined` are treated equally and the frontend never specifies `undefined`.

Reply:
Done in 9fae3e2535a1c52601085fc6126acc7d4efa034b